### PR TITLE
fix: site-footer-edits-fr

### PR DESCRIPTION
**Pull request by prycrane**
This is a text edit. I updated the pages to the new Jekyll template. @delisma - let's have a look at how the h1 is handled. Thanks.

- 	https://deploy-preview-131--systeme-conception-canada-ca.netlify.app/configurations-conception-communes/pied-page.html	
- 	https://deploy-preview-131--systeme-conception-canada-ca.netlify.app/configurations-conception-communes/pied-page-contextuelle.html
- 	https://deploy-preview-131--systeme-conception-canada-ca.netlify.app/configurations-conception-communes/pied-page-sous.html
- 	https://deploy-preview-131--systeme-conception-canada-ca.netlify.app/configurations-conception-communes/pied-page-principale.html

English pull request: https://github.com/canada-ca/design-system/pull/222

### CODE REVIEW

**Comment by delisma** [comment]
> This is a text edit. I updated the pages to the new Jekyll template. @delisma - let's have a look at how the h1 is handled. Thanks.

Yes waiting for PR #128 to be merged to have the H1 in production adopting the gc-thickline class.

**Review by LPipes4 — APPROVED** [review]

File: configurations-conception-communes/pied-page-contextuelle.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  </ul>
+  <h4>Mises en œuvre alternatives pour les pages transactionnelles et les pages de campagne</h4>
+  <ul>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/footers-fr.html">Pied de page complet (bande contextuelle, bande principale et bande sous de pied de page)</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-main-fr.html">Bande principale et bande sous pied de page sans les liens facultatifs</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footer-main-fr.html">Bande contextuelle et bande sous pied de page</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-contextual-fr.html">Bande contextuelle et bande sous pied de page sans les liens facultatifs</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-corporate-fr.html">Uniquement la bande sous pied de page</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footers-fr.html">Uniquement la bande sous pied de page sans liens facultatifs</a></li>
```
Comment (LPipes4) [inline configurations-conception-communes/pied-page-contextuelle.md:127]: Uniquement la bande sous pied de page sans les liens facultatifs

Comment (LPipes4) [reply configurations-conception-communes/pied-page-contextuelle.md:127]: just added "les" before liens

File: configurations-conception-communes/pied-page-principale.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+  </ul>
+  <h4>Mises en œuvre alternatives pour les pages transactionnelles et les pages de campagne</h4>
+  <ul>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/footers-fr.html">Pied de page complet (bande contextuelle, bande principale et bande sous de pied de page)</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-main-fr.html">Bande principale et bande sous pied de page sans les liens facultatifs</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footer-main-fr.html">Bande contextuelle et bande sous pied de page</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-contextual-fr.html">Bande contextuelle et bande sous pied de page sans les liens facultatifs</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-corporate-fr.html">Uniquement la bande sous pied de page</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footers-fr.html">Uniquement la bande sous pied de page sans liens facultatifs</a></li>
```
Comment (LPipes4) [inline configurations-conception-communes/pied-page-principale.md:138]: Uniquement la bande sous pied de page sans les liens facultatifs

File: configurations-conception-communes/pied-page-sous.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  </ul>
+  <h4>Mises en œuvre alternatives pour les pages transactionnelles et les pages de campagne</h4>
+  <ul>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/footers-fr.html">Pied de page complet (bande contextuelle, bande principale et bande sous de pied de page)</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-main-fr.html">Bande principale et bande sous pied de page sans les liens facultatifs</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footer-main-fr.html">Bande contextuelle et bande sous pied de page</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-contextual-fr.html">Bande contextuelle et bande sous pied de page sans les liens facultatifs</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-corporate-fr.html">Uniquement la bande sous pied de page</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footers-fr.html">Uniquement la bande sous pied de page sans liens facultatifs</a></li>
```
Comment (LPipes4) [inline configurations-conception-communes/pied-page-sous.md:138]: Uniquement la bande sous pied de page sans les liens facultatifs

File: configurations-conception-communes/pied-page.md [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+  </ul>
+  <h4>Mises en œuvre alternatives pour les pages transactionnelles et les pages de campagne</h4>
+  <ul>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/footers-fr.html">Pied de page complet (bande contextuelle, bande principale et bande sous de pied de page)</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-main-fr.html">Bande principale et bande sous pied de page sans les liens facultatifs</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footer-main-fr.html">Bande contextuelle et bande sous pied de page</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-contextual-fr.html">Bande contextuelle et bande sous pied de page sans les liens facultatifs</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-corporate-fr.html">Uniquement la bande sous pied de page</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footers-fr.html">Uniquement la bande sous pied de page sans liens facultatifs</a></li>
```
Comment (LPipes4) [inline configurations-conception-communes/pied-page.md:265]: Uniquement la bande sous pied de page sans les liens facultatifs

**Review by LPipes4 — CHANGES_REQUESTED** [review]

File: configurations-conception-communes/pied-page-principale.md (hunk 2 above)
Comment (prycrane) [reply configurations-conception-communes/pied-page-principale.md]: Done.  I love making the changes this way!

File: configurations-conception-communes/pied-page-sous.md (hunk 3 above)
Comment (prycrane) [reply configurations-conception-communes/pied-page-sous.md]: Done.

**Review by delisma — APPROVED** [review]

**Pull request closed without merging**